### ISSUE TITLE
Import .png files. Import button does nothing.

### ISSUE THREAD

**Issue by roteague** [mfractor/mfractor-feedback#313]
##Installation Information

```
--Platform Information--
Operating System: Windows
OS: Win32NT
OS Service Pack: 
OS Version: Microsoft Windows NT 10.0.19041.0
--MFractor Information--
Product Name: MFractor
Product Version: 4.4.0.0
Product Variant: Visual Studio For Windows
Product Variant Version: 16.0.0.0
Product SKU: MFRACTOR-VS-WINDOWS
Commit SHA: 456a155ffb822243ae788e1ff912fa28be4af2e3
Revision: 4600
Branch: master
Date: 06/08/2020 2:26:45 AM (UTC)
Agent: Local
Installed extensions:	
```

**Comment by matthewrdev** [comment]
@roteague I'm currently unable to reproduce this on my local machine (Visual Studio 2019 - 16.8.2, MFractor 4.4.3).

Can you try upgrading both Visual Studio and MFractor to their latest versions?

Also, can you expand on the "import button does nothing" comment? 

 1. Are you referring to the image importer or the app icon importer? 
 2. Are you able to open the image/app icon importer?
 3. How are you accessing the importer? Is this through the right-click Add menu on the project/solution or via the top MFractor menu?

**Comment by roteague** [comment]
I'm trying to Import an Image Asset. I have updated VS - 16.8.3, uninstalled and re-installed MFractor (Extensions manager shows 4.4.3, but MFractor reports 4.4.0). 

Import Image button does nothing.

--Platform Information--
Operating System: Windows
OS: Win32NT
OS Service Pack:
OS Version: Microsoft Windows NT 10.0.19041.0
--MFractor Information--
Product Name: MFractor
Product Version: 4.4.0.0
Product Variant: Visual Studio For Windows
Product Variant Version: 16.0.0.0
Product SKU: MFRACTOR-VS-WINDOWS
Commit SHA: 456a155ffb822243ae788e1ff912fa28be4af2e3
Revision: 4600
Branch: master
Date: 06/08/2020 2:26:45 AM (UTC)
Agent: Local
Installed extensions:    

----------------------------------------

**Comment by roteague** [comment]
Just upgraded to MFractor 4.4.4. Image still won't import. I'm able to select the image, it shows the different form sizes for the two platforms (iOS and Android), but the Import button is non-functional.

**Comment by ravero** [comment]
@roteague thanks for reporting this problem and sorry for the trouble. I've tracked the issue and fixed to be available on MFractor 4.4.5.

**Comment by roteague** [comment]
Thank you, that is greatly appreciated. I'll follow-up once release and I can test it.

Robert    

----------------------------------------

**Comment by matthewrdev** [comment]
@roteague This is now fixed and available in MFractor 4.4.5. Thanks for reporting the issue and mega-thanks to @ravero for fixing it 😁
